STAT302 - Assignment 2 Q 2.12

In [1]:
#Q2.12 a
beta1Hat = 5.3
beta1Se = 2.8
t = (beta1Hat - 0) / beta1Se
t

[1] 1.892857

In [2]:
PValue = 2 * (1 - pt(t, df = 80))
PValue

[1] 0.0619934

H0: beta1 = 0, Ha: beta1 /= 0 \
t = 1.89
PValue = 0.062

Failed to reject H0\
We have no evidence the slope is different from 0

In [3]:
#Q2.12 b
5.3 - 1.99 * 2.8
5.3 + 1.99 * 2.8

[1] -0.272

[1] 10.872

We have 95% confident the slope is between (-0.272, 10.872)